_opcodes/op005.html

```diff
@@ -24,3 +24,9 @@
 <br/>1003 Dire charmed (no text) (Target hostile after effect ends)
 <br/>1004 Controlled by Cleric (no text)
 <br/>1005 Target is hostile (no text)
+
+{% capture note %}
+If this effect turns a party member hostile, it will disable all of their scripts.
```

Comment:
Disable or reset to something? If they were just disabled, the party member would not attack you back.

Reply:
> Disable or reset to something? If they were just disabled, the party member would not attack you back.

It disables the scripts and then forces a specific action on party members without setting a new script, similar to how turn undead forces the Run Away action (on undead, not paladins).

Reply:
Is it AttackReevaluate(NearestEnemyOf(Myself), 1)? If it's known, it should be added here.

Reply:
> Is it AttackReevaluate(NearestEnemyOf(Myself), 1)? If it's known, it should be added here.

It's part of "CGameSprite::SpecialStateProcessing()"

If the creature is

1. STATE_CHARMED
2. Base EA <= GOODCUTOFF
3. Current EA == ENEMY

then it will:
`AttackReevaluate([GOODCUTOFF],10)`

That also clarifies the distinction of EA 254[EVILBYCHARM/CHARMED_PC] - it is not issued this attack action. Which is why "Normal" Charm doesn't cause allies to attack the party, while "Dire" Charm does.